sklearn/feature_extraction/text.py

```diff
@@ -1286,7 +1286,23 @@ def idf_(self, value):
 class TfidfVectorizer(CountVectorizer):
     """Convert a collection of raw documents to a matrix of TF-IDF features.
 
-    Equivalent to CountVectorizer followed by TfidfTransformer.
+    Equivalent to CountVectorizer followed by TfidfTransformer. 
+
+    CountVectorizer converts a collection of text documents to a matrix of token counts. 
+
+    TfidfTransformer then converts the count matrix from CountVectorizer to a normalized tf-idf representation. Tf is term frequency, and idf is inverse document frequency. This is a common way to calculate the count of a word relative to the appearance of a ducument. 
+
+    The formula that is used to compute the tf-idf of term t is
+    tf-idf(d, t) = tf(t) * idf(d, t), and the idf is computed as
```

Comment:
I think the other documentation has some formatting for these, can you make sure to copy the code, not the rendering?

Reply:
done!